README.md

```diff
@@ -1 +1,41 @@
-# lboard
\ No newline at end of file
+# lboard
+
+In this project, I have built a web app powered by the Leaderboard APIs. I've achieved by using the Webpack Framework.
+
+## Live Link
+
+[Live Link](https://softwareengineermwi.github.io/lboard/)
+
+## Project Requirements
+
+Implement only a basic UI with plain HTML:
+
+## Install
+
+- [Download](Clone the repository to your machine)
+
+- $ git clone https://github.com/softwareengineermwi/lboard/tree/dev
+
+- Navigate to the project folder after switching to the 'dev' branch and execute the liveServer. You can find this project on your browser where it will be hosted locally.
```

Comment:
### Please note that this is the final step to complete this project. So, it's so important to leave it professionally and technically completed 🙏 As I see, it's technically perfect 😉 Now let's work on the professional side 🥰

- [ ] At this step, please make sure to write descriptive step-by-step instructions on how to run the application locally. This is to help the non-technical to run your app easily on their local machines. 🙏 😄

**hint**
After they navigate to the repo's directory they will have to:

1. run `npm install` or `npm i` for installing necessary packages for running the app.
2. run `npm run build` for building the code files to the _dist_ dir.
3. run `npm run start` for the local host.